README.md

```diff
@@ -36,6 +36,8 @@ To run ACA-Py with this plugin locally, you'll need to provide the path to this
 aca-py start --plugin ./acapy_wallet_groups_plugin <other params>
 ```
 
+IMPORTANT NOTE: When passing an env file or env vars to the aca-py instance make sure to set `ACAPY_MULTITENANT_ADMIN=false` as opposed to `ACAPY_MULTITENANT_ADMIN=true`. If set to true the plugin will register and the endpoint will show up with the correct query fields in OpenAPI, _but_ under the hood not register the plugin correctly. That results in weird behaviour where no group_id key is returned in the response and querying by group_id just returns all wallets.
```

Comment:
Github now supports notes/warning blocks in Markdown: https://github.com/community/community/discussions/16925, maybe use the new warning note?

Reply:
Maybe we can also describe the config option / cli option `--multitenant-admin` for cli or `multitenant-admin: true` for yaml config